cdc/cdc/metrics.go

```diff
@@ -37,20 +36,12 @@ func init() {
 	kv.InitMetrics(registry)
 	puller.InitMetrics(registry)
 	sink.InitMetrics(registry)
-	entry.InitMetrics(registry)
 	processor.InitMetrics(registry)
 	keyspanpipeline.InitMetrics(registry)
 	owner.InitMetrics(registry)
 	etcd.InitMetrics(registry)
 	initServerMetrics(registry)
-	// actor.InitMetrics(registry)
 	orchestrator.InitMetrics(registry)
 	p2p.InitMetrics(registry)
-	// Sorter metrics
-	// sorter.InitMetrics(registry)
-	// memory.InitMetrics(registry)
-	// unified.InitMetrics(registry)
-	// leveldb.InitMetrics(registry)
-	// redowriter.InitMetrics(registry)
```

Comment:
how about just removing them instead of commenting them out?

Reply:
yes, we should remove them.

Reply:
yes, we should remove them.